src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,24 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        if (!err && dependency._package.bin) {
+          const srcBinLocation = path.join(linkSrc, dependency._package.bin);
```

Comment:
Would be nice to add a `bin` getter in src/Package.js for consistency. Then this could be:

```javascript
const srcBinLocation = path.join(linkSrc, dependency.bin);
```